src/Microsoft.Identity.Client/Internal/Telemetry/TelemetryTokenCacheAccessor.cs

```diff
@@ -36,56 +34,56 @@ internal class TelemetryTokenCacheAccessor : TokenCacheAccessor
         public void SaveAccessToken(string cacheKey, string item, RequestContext requestContext)
         {
             var cacheEvent = new CacheEvent(CacheEvent.TokenCacheWrite) { TokenType = CacheEvent.TokenTypes.AT };
-            Telemetry.GetInstance().StartEvent(requestContext.TelemetryRequestId, cacheEvent);
+            Client.Telemetry.GetInstance().StartEvent(requestContext.TelemetryRequestId, cacheEvent);
```

Comment:
>Client [](start = 12, length = 6)

You could also have "using Microsoft.Identity.Client;"... That way you don't need to do Client.Telemetry